Description: Restore TEXTDOMAINDIR correction
 Restore TEXTDOMAINDIR correction from r1889, lost apparently by mistake in
 r2156.  Noticed by Anthony Fok.
Origin: upstream, http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/2326
Forwarded: yes
Last-Update: 2010-04-14

diff -Nur -x '*.orig' -x '*~' grub2/util/grub.d/10_kfreebsd.in grub2.new/util/grub.d/10_kfreebsd.in
--- grub2/util/grub.d/10_kfreebsd.in	2010-03-08 19:34:05.000000000 +0000
+++ grub2.new/util/grub.d/10_kfreebsd.in	2010-04-14 14:29:31.000000000 +0100
@@ -23,7 +23,7 @@
 . ${libdir}/grub/grub-mkconfig_lib
 
 export TEXTDOMAIN=@PACKAGE@
-export TEXTDOMAINDIR=@LOCALEDIR@
+export TEXTDOMAINDIR=@localedir@
 
 CLASS="--class os"
 
diff -Nur -x '*.orig' -x '*~' grub2/util/grub.d/10_linux.in grub2.new/util/grub.d/10_linux.in
--- grub2/util/grub.d/10_linux.in	2010-04-14 14:27:31.000000000 +0100
+++ grub2.new/util/grub.d/10_linux.in	2010-04-14 14:29:31.000000000 +0100
@@ -23,7 +23,7 @@
 . ${libdir}/grub/grub-mkconfig_lib
 
 export TEXTDOMAIN=@PACKAGE@
-export TEXTDOMAINDIR=@LOCALEDIR@
+export TEXTDOMAINDIR=@localedir@
 
 CLASS="--class gnu-linux --class gnu --class os"
 
